qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst

```diff
@@ -18,6 +26,12 @@ You can download an example sample information file and prep information file fr
 Sample information file
 -----------------------
 
+Similar to a `QIIME metadata file, <http://qiime.org/documentation/file_formats.html#metadata-mapping-files>`__
```

Comment:
Thanks for working with me on this. I absolutely agree that documentation is the linchpin for usable software. I'm new to writing it, so your feedback is essential. Thanks!